### PR TITLE
✨ NEW: Add RTD site

### PR DESCRIPTION
**Pull request by chrisjsewell**
Because why not!

 - [ ] Used [AEP template](../blob/master/000_aep_guidelines/readme.md) from AEP 0
 - [ ] Status is `submitted`
 - [ ] Added type & status labels to PR
 - [ ] Added AEP to `README.md`
 - [ ] Provided github handles for authors

### CODE REVIEW

**Comment by sphuber** [comment]
Nice. Should we maybe put the doc-specific files in a `docs` sub directory to keep root from getting to cluttered?

**Comment by chrisjsewell** [comment]
> Should we maybe put the doc-specific files in a docs sub directory to keep root from getting to cluttered?

Ah but then I believe you would have to put all the documentation under the `docs` directory, which is obviously everything in the repository lol

**Comment by chrisjsewell** [comment]
There you go: https://aep--33.org.readthedocs.build/en/33/

**Comment by chrisjsewell** [comment]
All done on my end.
Note, when you add a new AEP, you just need to add it to `_toc.yml`

**Comment by sphuber** [comment]
> Note, when you add a new AEP, you just need to add it to `_toc.yml`

Would be good to add this to the steps for submitting an AEP in AEP 000.

File: _toc.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+root: README.md
+subtrees:
+- caption: Proposals 
+  entries:
```
Comment (ltalirz) [inline _toc.yml:4]: There's no way to automate this, right?

File: 000_aep_guidelines/readme.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ A summary for pros and cons for the proposed enhancement, detailing e.g. which p
    and put readme file in [Markdown](https://guides.github.com/features/mastering-markdown/) format inside this folder.
      * You may use this `readme.md` as a template.
      * You may use the folder to include additional files relevant to your enhancement proposal.
-     * Add your AEP to the `README.md` at the top level of the repository.
+     * Add your AEP to the `README.md` at the top level of the repository, and in to `_toc.yml`.
```
Comment (chrisjsewell) [inline 000_aep_guidelines/readme.md:77]: is that ok?

**Review by ltalirz — APPROVED** [review]
Thanks @chrisjsewell !

>> Should we maybe put the doc-specific files in a docs sub directory to keep root from getting to cluttered?

> Ah but then I believe you would have to put all the documentation under the docs directory, which is obviously everything in the repository lol

I would also have preferred that but if there's no way to do a `../` there, it's ok with me.

File: 000_aep_guidelines/readme.md (hunk 2 above)
Comment (sphuber) [reply 000_aep_guidelines/readme.md:77]:
```suggestion
     * Add your AEP to the `README.md` and `_toc.yml` files at the root of the repository.
```

**Review by sphuber — APPROVED** [review]

File: _toc.yml (hunk 1 above)
Comment (chrisjsewell) [reply _toc.yml:4]: nope, not currently

**Pull request merged**